documentation/openapi.json

```diff
@@ -2053,7 +2061,8 @@
           }
         },
         "additionalProperties": false,
-        "description": "Represents the details of a given process used in an operation."
+        "description": "Represents the details of a given process used in an operation.",
+        "nullable": true
```

Comment:
Do you know why some of these object schemas have `"nullable": true` on them? I can rationalize the one on OperationError (because all of its properties are nullable), but I can't for this schema. It feels strange to me that an object schema itself would declare that it is nullable rather than the usage of the schema would also allow null.